src/target/riscv/riscv.c

```diff
@@ -3748,31 +3745,52 @@ COMMAND_HANDLER(riscv_set_enable_virt2phys)
 
 COMMAND_HANDLER(riscv_set_ebreakm)
 {
-	if (CMD_ARGC != 1) {
-		LOG_ERROR("Command takes exactly 1 parameter");
+	struct target *target = get_current_target(CMD_CTX);
+	RISCV_INFO(r);
+
+	if (CMD_ARGC == 0) {
+		command_print(CMD, "riscv_ebreakm enabled = %d", r->riscv_ebreakm);
```

Comment:
Optionally - if you like:

```suggestion
		command_print(CMD, "riscv_ebreakm enabled = %s", r->riscv_ebreakm ? "yes" : "no");
```

Reply:
I think, that now it much easier to parse output of this command than with your suggestion, so I would like to leave it as it is.

Reply:
Let's follow existing OpenOCD commands. E.g. handle_poll_command() prints out a boolean value, and it uses:
`command_print(CMD, "background polling: %s", jtag_poll_get_enabled() ? "on" : "off");`

Reply:
Addressed